db/data/movies.js

```diff
@@ -0,0 +1,19 @@
+const apiCalls = require("../../apiCalls.js")
+
+const movieData = async (genre) => {
+  const moviesByGenre = await apiCalls.movieFetch(genre)
+  let moviesToPost = []
+  let counter = 0;
+  while (moviesToPost.length < 25) {
+    let randomMovieTitle = moviesByGenre[counter].split('/')[2]
```

Comment:
At this point on line 8, would you be able to get the `randomMovieDetails` based on the `randomMovieTitle`?  In other words, doesn't the movie's title come from an object that has the rest of the related key-value pairs?

Reply:
Yeah so the way this API is set up is a little different, and the path I chose was a little more complex. 

`moviesByGenre` as seen on line 8 is actually the array containing the 100 top movies for a specific genre. These top 100 movies are actually just strings with a unique key that represent a path on a separate endpoint.

Line 8 is pulling the key from the array at a specified index so that it can be used as a parameter by the `singleMovie` apiCall. This call is where all the data we actually need to house in our database will come from, so that we may relay this information to our user. 

See the below screenshots for a visual representation of this:

What `moviesByGenre` stores: 

![Screen Shot 2020-10-28 at 9 27 08 PM](https://user-images.githubusercontent.com/65369275/97514104-577a7800-1964-11eb-8bd4-3cd57d17e99b.png)


What `randomMovieDetails` stores: 

![Screen Shot 2020-10-28 at 9 26 21 PM](https://user-images.githubusercontent.com/65369275/97514063-3b76d680-1964-11eb-8274-641a44f897f6.png)